test/html/test_document_encoding.rb

```diff
@@ -3,8 +3,8 @@
 
 module Nokogiri
   module HTML
-    if RUBY_VERSION =~ /^1\.9/
-      class TestDocumentEncoding < Nokogiri::TestCase
+    class TestDocumentEncoding < Nokogiri::TestCase
+      if RUBY_VERSION =~ /^1\.9/  
```

Comment:
I imagine, given the time frame of the submission, that this was intended to exclude 1.8 and not to exclude 2.0+. Correct?